uiowa/uiowa
# Feature: card grid fixes

Resolves https://github.com/uiowa/uiowa/issues/3140 and resolves https://github.com/uiowa/uiowa/issues/3145.

# How to test
`yarn workspace uids_base gulp --development`

1. Using the events block using the grid option on a `v3` site, change the card styles to "stacked" with "hide images" and verify that the top padding even among each card. 
2. Using the events block using the grid/masonry option on a `v2` site, hide the images and verify that the top padding is equal among all the cards. 
3. Place a list grid into a `Page w/ left sidebar` layout with no content in the left sidebar.  Save the page and verify that the grid will appear.   
4. Place a list grid into a `Page w/ left sidebar` layout with a left sidebar menu.  Hide the sidebar from rendering on the node edit screen and verify that the list grid still appears in a grid.
